src/solvers/flattening/boolbv_abs.cpp

```diff
@@ -16,24 +16,17 @@ Author: Daniel Kroening, kroening@kroening.com
 
 bvt boolbvt::convert_abs(const exprt &expr)
 {
-  std::size_t width=boolbv_width(expr.type());
-
+  const std::size_t width = boolbv_width(expr.type());
   if(width==0)
     return conversion_failed(expr);
-
```

Comment:
Please stop removing blank lines that clearly separate steps. Code readability matters. Really.